packages/examples/examples/wasm/src/index.js

```diff
@@ -0,0 +1,42 @@
+const { ethErrors } = require('eth-rpc-errors');
+const { PROGRAM_WASM_HEX } = require('../build');
+
+// Ref:
+// - https://developer.mozilla.org/en-US/docs/WebAssembly/Using_the_JavaScript_API
+// - https://github.com/mdn/webassembly-examples/tree/06556204f687c00a5d9d3ab55805204cbb711d0c/js-api-examples
+
+let wasm;
+
+const initializeWasm = async () => {
+  try {
+    const wasmBuffer = arrayBufferFromHex(PROGRAM_WASM_HEX);
+    wasm = await WebAssembly.instantiate(wasmBuffer);
+  } catch (error) {
+    console.error('Failed to initialize WebAssembly module.', error);
+    throw error;
+  }
+};
+
+wallet.registerRpcMessageHandler(async (_originString, requestObject) => {
+  if (!wasm) {
+    await initializeWasm();
+  }
+
+  switch (requestObject.method) {
```

Comment:
we should just pass the method/params through to be called on wasm if it exists:

```
  if (wasm.instance.exports[requestObject.method]) {
    return wasm.instance.exports[requestObject.method](...requestObject.params);
  }
```

Reply:
Sure 👍

Reply:
Done!